app/views/posts/_postform.html.erb

```diff
@@ -1,8 +1,13 @@
 <div class="big_form">
 <div class="new_post_container">
+  <!-- mms: is you used the object here, rails would take care of create/update
+  form_for @post do |f| -->
 <%= form_for :post, url: posts_path do |f| %>
 
   <%= f.label "On which metro-line did this occur?"%><br>
+    <!-- mms: lots of duplication here.  I recommend a PORO (Plain-ole-ruby-object) model.
```

Comment:
Hi Matt,
Thanks for all of your feedback. My form sure is repetitive. The reason I did it was because I needed to re-arrange the radio buttons for smaller screens. There's definitely a better way to doing this but this was the only solution I could come up with.